package.json

```
@@ -83,8 +90,11 @@
   "types": "index.d.ts",
```

Types value is wrong
```suggestion
  "types": "dist/index.d.ts",
```

---

Oh my god I think I got it now 💡 

https://github.com/faker-js/faker/blob/18ab2c7b3c8fa78c46d4abafc07c3219f41caa46/package.json#L86

previously we had `*` here, but now we use `.`
so previously the `*` was put in front of everything, but the `.` is now a relative start from here
and this is why `dist/` is now required and fixes the current problem